src/tests/tests/utils.js

```diff
@@ -515,14 +515,38 @@ QUnit.test("estimateMaxEntropy", assert => {
 
   assert.notEqual(
     utils.estimateMaxEntropy('google.com/analytics'),
-    utils.estimateMaxEntropy('0191/_-google9fjkelo'),
+    utils.estimateMaxEntropy('0191/_-goo~le9x+xzxo'),
     "strings of the same length but from different character classes will estimate different entropy values"
   );
 
   assert.equal(
     utils.estimateMaxEntropy("google.com/0191/_-google/analytics.fizz?buzz=foobar"),
     320.55551316197466,
-    "entropy for complex string of varying character classes estimates entropy correctly"
+    "entropy for complex string of varying character classes is correctly estimated"
+  );
+
+  assert.equal(
+    utils.estimateMaxEntropy("03899029.01_293"),
+    49.82892142331044,
+    "entropy for string from the common classes of characters is correctly estimated"
+  );
+
+  assert.equal(
+    utils.estimateMaxEntropy("fizzBUZZ012345"),
+    84,
+    "entropy for string from the case-insensitive class of characters is correctly estimated"
+  );
+
+  assert.equal(
+    utils.estimateMaxEntropy("fizz/buzz+fizzy~buzzy%"),
+    142.82076811925285,
+    "entropy for string from the case-sensitive class of characters is correctly estimated"
+  );
+
+  assert.equal(
+    utils.estimateMaxEntropy("1280x720") < 32,
+    true,
+    "resolution strings with 'x' char from SEPS class are correctly estimated as low entropy"
```

Comment:
We don't seem to test the "If all characters are upper or lower case, don't consider case when computing entropy." scenario.

Reply:
Isn't that taken care of in [this](https://github.com/EFForg/privacybadger/blob/88fdf9fd9a6b357fc38450f2e5cd9b1e434c7d48/src/tests/tests/utils.js#L504) ?

Reply:
Unless you think it's worth having a test that compares a string of mixed cases with one that has the same characters but in all one case?

Reply:
If I remove the relevant code (below), the tests still pass for some reason.
```javascript
  let sameCase = (str.toLowerCase() == str) || (str.toUpperCase() == str);
  if (sameCase) {
    str = str.toLowerCase();
  }
```

Reply:
Ah I see, in the test I mentioned, the `. / ` URL characters make the strings fall into the `SEPS` and `B64` character classes. I modified the test so that it depends on the relevant code block to pass.